pegout-request.js

```diff
@@ -10,12 +10,41 @@ class PegoutRequest {
     }
 }
 
+function validatePegoutRequestBufferFields(
+    destinationAddressHash160Buffer,
+    amountInSatoshisBuffer,
+    rskTxHashBuffer
+) {
+    if (destinationAddressHash160Buffer.length !== 20)
+        return false;
+
+    if (rskTxHashBuffer.length !== 32)
+        return false;
+
+    return true;
```

Comment:
```suggestion
    return destinationAddressHash160Buffer.length === 20 && rskTxHashBuffer.length === 32;
```
Just a suggestion, if you prefer a shorter version

Reply:
```suggestion
    if(destinationAddressHash160Buffer.length !== 20 || rskTxHashBuffer.length !== 32) {
     throw new Error("....");
    }
```
Another option in case we decide to throw an error for invalid data

Reply:
> Just a suggestion, if you prefer a shorter version

You're right. I did it in that way thinking in readability and thinking if maybe tomorrow there is a more complex validation to be checked, in one line you would need to split it as soon as that line starts being too long or difficult to understand.

But I can go with your suggestion, and if in the future the long version is more appropriate for readability so we do it in that way.

>Another option in case we decide to throw an error for invalid data

I thought about that but I considered that doing it may break the code of the consumers of the library. The consumer will need to manage the exception when using the new version of the parser library.

For this reason, I just left the logic as I found it, returning an empty array when parsing the input is not possible.

Reply:
Ok, thanks for your answer. I'd still prefer to throw an error if the received data is not valid. I'll let @josedahlquist and @kelvinator07 chime in and share their thoughts. Thanks!

Reply:
Me too. I think if validation fails for these inputs, this is a nonrecoverable issue and we should throw an error.